backend/views.py

```diff
@@ -1,8 +1,17 @@
+import json
 from django.shortcuts import render
-from django.http import HttpResponse
+from django.http import HttpResponse, JsonResponse
+from .models import *
 
 # Create your views here.
 
 def home(request):
     file = open('static/index.html').read()
     return HttpResponse(file)
+
+def getTasks(request):
+    tasks = Task.objects.all()
+    taskDict = {'tasks': []}
+    for x in tasks:
+        taskDict['tasks'].append(x.title)
```

Comment:
This works but could be written in a shorter way. Have you looked into the [values](https://docs.djangoproject.com/en/4.1/ref/models/querysets/#values) method? Especially the variant with passing arguments - the second example in linked documentation should be helpful.